src/ParametersGenerator.php

```diff
@@ -12,7 +12,7 @@ public function generate(string $sql): array
     {
         // Split SQL to statements
         $statements = array_map(function ($sql) {
-            return SqlFormatter::format($sql, false);
+            return trim($sql);
```

Comment:
Formatter mi strasne rozbijal SQLko, ... komentare/joiny ... preto som ho vypol, v Twig-u sa to da naformatovat ako potrebujeme.

Reply:
jj on ho občas pěkně rozbije... přes twig je to lepší řešení :+1: